ceph_salt/config_shell.py

```diff
@@ -8,7 +8,16 @@
 import string
 
 
-from pyparsing import alphanums, OneOrMore, Optional, Regex, Suppress, Word, QuotedString
+from pyparsing import (
+    alphanums,
+    OneOrMore,
+    Optional,
+    Regex,
+    Suppress,
+    Word,
+    QuotedString,
+    alphas8bit
```

Comment:
This is just a nit from an "alphabetical-ordering-in-code freak", so take it with a grain of salt. The list seems to be in alphabetical order all the way up to "Word", so would it make sense to move "QuotedString" and "alphas8bit" so the whole list is sorted?

Reply:
Sure, I completely agree. Done.